go.mod

```diff
@@ -4,42 +4,43 @@ go 1.21.3
 
 require (
 	github.com/stretchr/testify v1.9.0
-	github.com/testcontainers/testcontainers-go v0.30.0
+	github.com/testcontainers/testcontainers-go v0.31.1-0.20240524121510-d4a21ea92ee8
```

Comment:
⚠️ We're waiting for v0.31.0, blocked by release of fix https://github.com/testcontainers/testcontainers-go/pull/2551